src/components/NcRichText/NcRichText.vue

```diff
@@ -449,24 +471,37 @@ export default {
 
 						if (!tag.startsWith('#')) {
 							if (this.useExtendedMarkdown) {
+								let nestedList
```

Comment:
As it could be not only list:

```suggestion
								let nestedNode = null
```